requirements.txt

```diff
@@ -1,13 +1,13 @@
 absl-py==0.15.0
-aiohttp==3.8.0
+aiohttp==3.8.1
```

Comment:
Would `mac_requirements.txt` also change?

Reply:
@RutvikGupta would you be able to comment on this?

Reply:
I believe it should be updated as well.

Reply:
Automated update of `mac_requirements.txt` for mac is included in pr #1124.